rujitsu.gemspec

```diff
@@ -23,4 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.12"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "timecop"
```

Comment:
Using Timecop is breaking the builds on Travis for Ruby 1.8.7 - https://travis-ci.org/brightbox/rujitsu

That is probably acceptable in 2017 so can you also update the Travis build matrix to allow failures?